library/src/main/java/com/novoda/downloadmanager/NotificationDispatcher.java

```diff
@@ -45,11 +44,12 @@ void updateNotification(DownloadBatchStatus downloadBatchStatus) {
 
             if (status == DOWNLOADED) {
                 notificationSeenPersistence.updateNotificationSeenAsync(downloadBatchStatus.getDownloadBatchId(), NOTIFICATION_SEEN);
+            }
+
+            if (notificationInformation.notificationStackState() == STACK_NOTIFICATION_DISMISSIBLE) {
```

Comment:
Since this is an enum now, wouldn't it be better to use `switch` here?

Reply:
edaaa50